tests/data/ar/Animal.php

```diff
@@ -38,10 +38,10 @@ public function getDoes()
 
     /**
      *
-     * @param type $row
+     * @param array $row
      * @return \yiiunit\data\ar\Animal
      */
-    public static function instantiate($row)
+    public static function instantiate($row = [])
```

Comment:
@Faryshta could you please provide an example of code which will break after this change?